lessons/welldone.md

```diff
@@ -1,15 +1,15 @@
 ## Well done
 
-You have successfully completed the codelab and learnt how to build an awesome app.
+You have successfully completed the codelab and learnt how to build a Firefly React app using fundamental React knowledge and React Spectrum components.
 
 ### Achievements
 
 You have learned: 
 
-* A
-* B
-* C
+* How to use [aio-lib-state](https://github.com/adobe/aio-lib-state) as part of Runtime actions.
+* How to bind React Spectrum components to Runtime actions.    
 
 ### What's next ?
 
-Stay tuned.
\ No newline at end of file
+Type checking React properties is already possible in JavaScript using `prop-types`. 
+If you want to go further and write Typed JavaScript inside a Firefly App for your Runtime actions and React app, we'll show you how to in the upcoming codelab **Using TypeScript to develop a Project Firefly app**.    
```

Comment:
Although it's nice to have the typescript codelab, not sure if we should proactively promote it here (it's not yet an officially supported part of Firefly). Probably showing the generic message "visit other codelabs" is good enough.

Reply:
Yup sounds good to me.